sentry-raven.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "faraday", ">= 0.7.6"
   gem.add_dependency "certifi"
+  gem.add_dependency "binding_of_caller" if RUBY_VERSION >= '1.9.2' && !defined?(JRUBY_VERSION)
```

Comment:
I think we should remove this line, and instead, people should be instructed to add "binding_of_caller" to their Gemfile if they want to use this feature. Similar to how `dalli` uses `kgio`.